example-get-started/generate.sh

```diff
@@ -5,16 +5,16 @@
 # x Print commands and their arguments as they are executed.
 set -eux
 
-THIS="$( cd "$(dirname "$0")" ; pwd -P )"
+HERE="$( cd "$(dirname "$0")" ; pwd -P )"
 REPO_NAME="example-get-started"
-REPO_PATH="../$REPO_NAME"
+REPO_PATH="$HERE/build/$REPO_NAME"
```

Comment:
add build to gitignore?

Reply:
Good thinking. `build/` is already in the root .gitignore but that was a coincidence. Will add the the directory .gitignore too.

Reply:
Added in 1a9ce81 (will show up in my next PR).